scripts/jenkins/ardana/ansible/roles/reboot_node/tasks/reboot_cloud_nodes_crowbar.yml

```diff
@@ -0,0 +1,31 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+
+- name: Get list of nodes
+  shell: |
+    crowbar_node_state status | awk '/^d/{print $1}'
+  register: _list_of_crowbar_nodes
+
+
```

Comment:
this task is named `reboot_cloud_nodes_...`, but there is no task here that actually reboots the nodes. Maybe you forgot a step ?

Reply:
Good catch, I accidentally deleted it during one debugging session. :'(